pkg/util/log/log.go

```diff
@@ -771,3 +785,8 @@ func ChangeLogLevel(l seelog.LoggerInterface, level string) error {
 	// need to return something, just set to Info (expected default)
 	return errors.New("cannot change loglevel: logger not initialized")
 }
+
+// isServerless returns whether or not the agent is running in a serverless context
+func isServerless() bool {
+	return len(os.Getenv("AWS_LAMBDA_FUNCTION_NAME")) > 0
```

Comment:
You could consider leveraging the `serverless` tag instead here.

Having a `isServerless()` implementation returning `false` when the `serverless` tag has not been set on build etc.

Not a strong opinion.

Reply:
Good idea, I've just updated the PR